docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md

```diff
@@ -1,25 +1,24 @@
-# Converting a PaddlePaddle* Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Paddle}
+# Converting a PaddlePaddle Model <a name="Convert_From_Paddle"></a> {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Paddle}
```

Comment:
I'm not sure what the link tag does here but I would love to see it gone... I would also be careful with using additional tags in the first-level header, we don't know if sphinx likes it :) Sometimes it may, other times, not so much :D